### DISCUSSION TITLE
Defaults for HTML to VanJS converter

### DISCUSSION THREAD

**Discussion by HEAVYPOLY** [vanjs-org/van#319] [Ideas]
I'm always toggling these 2 options on for the HTML to vanJS converter, would it make sense for them to be default on?
<img width="829" alt="image" src="https://github.com/vanjs-org/van/assets/24432677/497a1f2c-08e0-4e4c-8c23-5735748de0a9">

**Comment by sirenkovladd** [comment]
I suggest add these parameters to the local storage

**Reply by sirenkovladd** [reply]
https://github.com/vanjs-org/vanjs-org.github.io/pull/7

**Comment by Tao-VanJS** [comment]
Thanks @sirenkovladd, I merged your PR and also made indent saved in `localStorage` as well.
